packages/trader/src/Modules/Trading/Components/Elements/mobile-widget.jsx

```diff
@@ -1,28 +1,17 @@
 import React from 'react';
 import { Money } from '@deriv/components';
-import { localize, Localize } from '@deriv/translations';
+import { localize } from '@deriv/translations';
 import { getExpiryType, getDurationMinMaxValues, getLocalizedBasis } from '@deriv/shared';
 import { MultiplierAmountWidget } from 'Modules/Trading/Components/Form/TradeParams/Multiplier/widgets.jsx';
 import TradeParamsModal from '../../Containers/trade-params-mobile';
 import { observer, useStore } from '@deriv/stores';
 import { useTraderStore } from 'Stores/useTraderStores';
 
-const MobileWidget = observer(({ toggleDigitsWidget, is_collapsed }) => {
+const MobileWidget = observer(() => {
     const { ui } = useStore();
     const { onChangeUiStore } = ui;
     const trade_store = useTraderStore();
-    const {
-        amount,
-        basis,
-        currency,
-        duration,
-        duration_min_max,
-        duration_unit,
-        form_components,
-        is_multiplier,
-        last_digit,
-        onChange,
-    } = trade_store;
+    const { amount, basis, currency, duration, duration_min_max, duration_unit, is_multiplier, onChange } = trade_store;
```

Comment:
`toggleDigitsWidget` + `is_collapsed` from `props` and `form_components` + `last_digit` from `trade_store` were removed as they were needed for `Digit Widget`. As `Digit Widget` was removed to the _packages/trader/src/Modules/Trading/Components/Form/screen-small.tsx_ file, there is no need in them here.